docs/user/metal/install_ipi.md

```diff
@@ -195,3 +195,82 @@ network configuration, but fully supporting alternative subnets for the
 provisioning network is incomplete.
 
 https://github.com/openshift/installer/issues/2091
+
+## Troubleshooting
+
+General troubleshooting for OpenShift installations can be found
+[here](/docs/user/troubleshooting.md).
+
+### Bootstrap
+
+The bootstrap VM by default runs on the same host as the installer. This
+bootstrap VM runs the Ironic services needed to provision the control
+plane. Ironic being available is dependent on having successfully
+downloaded the machine OS and Ironic agent images. In some cases, this
+may fail, and the installer will report a timeout waiting for the Ironic
+API.
+
+To login to the bootstrap VM, you will need to ssh to the VM using the
+`core` user, and the SSH key defined in your install config.
+
+The VM obtains an IP address from your DHCP server on the external
+network. When using a development environment with
+[dev-scripts](https://github.com/openshift-metal3/dev-scripts), it uses
+the `baremetal` libvirt network unless an override is specified. The IP
+can be retrieved with `virsh net-dhcp-leases baremetal`. If the install
+is far enough along to have brought up the provisioning network, you may
+use the provisioning bootstrap IP which defaults to 172.22.0.2.
+
+Viewing the virtual machine's console with virt-manager may also be
+helpful.
+
+You can view the Ironic logs by sshing to the bootstrap VM, and
+examining the logs of the `ironic` service, `journalctl -u ironic`. You
+may also view the logs of the individual containers:
+
+  - `podman logs ipa-downloader`
+  - `podman logs coreos-downloader`
+  - `podman logs ironic`
+  - `podman logs ironic-inspector`
+  - `podman logs ironic-dnsmasq`
```

Comment:
the gather bootstrap already captures all the logs from pods, containers and other things if the bootstrapping fails.
I think it is more helpful to provide troubleshooting documents using the standard patterns instead of creating new steps of SSHing in.

Reply:
Well, install gather does not work on baremetal yet (#2009), I have a branch in progress to do it, but some things aren't complete as we need to be able to get the IP's for the control plane. 

Also, at least on baremetal platform the way things are today, there is a lot of value of ssh'ing to the bootstrap to review the individual logs of Ironic to troubleshoot what's gone wrong, and understanding all the containers that are running. I think it should stay, and remove it when #2009 is addressed.

And, since we're talking about install gather, I discovered some things in the progress of working on support in baremetal and broke some clean-ups out into their own PR which needs review
https://github.com/openshift/installer/pull/2698

Reply:
Well, install gather does not work on baremetal yet (#2009), I have a branch in progress to do it, but some things aren't complete as we need to be able to get the IP's for the control plane.

Also, at least on baremetal platform the way things are today, there is a lot of value of ssh'ing to the bootstrap to review the individual logs of Ironic to troubleshoot what's gone wrong, and understanding all the containers that are running. I think it should stay, and remove it when #2009 is addressed.

And, since we're talking about install gather, I discovered some things in the progress of working on support in baremetal and broke some clean-ups out into their own PR which needs review
#2698

Reply:
> Well, install gather does not work on baremetal yet (#2009), I have a branch in progress to do it, but some things aren't complete as we need to be able to get the IP's for the control plane.
> 
> Also, at least on baremetal platform the way things are today, there is a lot of value of ssh'ing to the bootstrap to review the individual logs of Ironic to troubleshoot what's gone wrong, and understanding all the containers that are running. I think it should stay, and remove it when #2009 is addressed.

This useless to us when we debug our CI runs, because we can't SSH while the test is running, so I don't think there is value in letting users so that too.

Making sure you can trace the bugs or what went wrong from gathered logs after failure is important requirement. and forcing it makes sure we are not loosing logs from containers that are only accessible when running.

Reply:
I really do get it, I understand the importance of gathering the information in the right way, but I also waste a lot of my day giving these exact troubleshooting steps to people on Slack.

If you want to revert this, I'll just move it somewhere else as it has to exist somewhere. I'd really rather it stay until I can give you install-gather support for the platform.

Reply:
> I really do get it, I understand the importance of gathering the information in the right way, but I also waste a lot of my day giving these exact troubleshooting steps to people on Slack.
> 
> If you want to revert this, I'll just move it somewhere else as it has to exist somewhere. I'd really rather it stay until I can give you install-gather support for the platform.

I wasn't making the case for reverting, rather documenting running the gather command specifying the ip addresses for bootstrap and control-plane machines manually _for now_ and then documenting how to scrub and look for failure in the gather log-bundle.

Also, I don't think this needs to belong anywhere else, it is good in the installer repo, more so  not documenting debugging workflows that are not useful debugging CI runs.